crates/project-model/src/sysroot.rs

```diff
@@ -115,10 +115,19 @@ impl Sysroot {
         Ok(Sysroot::load(sysroot_dir, src))
     }
 
-    pub fn discover_rustc(&self) -> Option<ManifestPath> {
+    pub fn discover_rustc_src(&self) -> Option<ManifestPath> {
```

Comment:
This name appeared to be a little inaccurate given its usage, but happy to undo.